src/openforms/authentication/static_variables/static_variables.py

```diff
@@ -61,8 +72,7 @@ class AuthKvK(BaseStaticVariable):
     name = _("Authentication KvK")
     data_type = FormVariableDataTypes.string
 
-    @staticmethod
```

Comment:
Nice, I had this on my list of TODO cleanups

Reply:
I hate these type error though, semantically this *is* perfectly fine to be a static method.